### ISSUE TITLE
The LR scheduler is not saved to (or loaded from) checkpoint

### ISSUE THREAD

**Issue by Nash2325138** [amjltc295/pytorch-golden-template#50]
As stated in the title, the saving and loading of state_dict of lr_scheduler are not handled.
However, our learning rate when resuming from a checkpoint is still correct, which makes me pretty confused about why...
